php/async/Exchange.php

```diff
@@ -53,7 +53,7 @@ class Exchange extends \ccxt\Exchange {
     public $marketsLoading = null;
     public $reloadingMarkets = null;
     public $tokenBucket;
-    public Throttler $throttler;
```

Comment:
temporarily removed, because in sync base there is no type `Throttler` and for now, i've simplified the process. however, i'll do something in separate PR thereafter to place this type back